db/migrate/20230919194135_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.belongs_to :user, foreign_key: { to_table: :users }, index: true
+      t.string :title
+      t.string :text
+      t.integer :comments_counter
+      t.integer :liked_counter
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] According to the ERD given:

![image](https://github.com/ahmedeid6842/Blog_Rails/assets/77069011/27bc4928-5b14-414f-9522-bd757ed996b6)

The Post should belong to an `Author`. To achieve this, you can use the following change:

```suggestion
class CreatePosts < ActiveRecord::Migration[7.0]
  def change
    create_table :posts do |t|
      t.belongs_to :author, foreign_key: { to_table: :users }, index: true
      t.string :title
      t.string :text
      t.integer :comments_counter
      t.integer :liked_counter
      t.timestamps
    end
  end
end
```

Now, each Post is going to have an attribute `author` that acts as a more descriptive alias for the user! 🤓 🔧